api/metric/gauge.go

```diff
@@ -56,46 +64,66 @@ func newGauge(name string, valueKind ValueKind, mos ...GaugeOptionApplier) Commo
 	return m
 }
 
+// NewFloat64Gauge creates a new gauge for float64.
 func NewFloat64Gauge(name string, mos ...GaugeOptionApplier) (g Float64Gauge) {
 	g.CommonMetric = newGauge(name, Float64ValueKind, mos...)
 	return
 }
 
+// NewInt64Gauge creates a new gauge for int64.
 func NewInt64Gauge(name string, mos ...GaugeOptionApplier) (g Int64Gauge) {
 	g.CommonMetric = newGauge(name, Int64ValueKind, mos...)
 	return
 }
 
+// GetHandle creates a handle for this gauge. The labels should
+// contain the keys and values specified in the gauge with the
+// WithKeys option.
 func (g *Float64Gauge) GetHandle(labels LabelSet) (h Float64GaugeHandle) {
 	h.Handle = g.getHandle(labels)
 	return
 }
 
+// GetHandle creates a handle for this gauge. The labels should
+// contain the keys and values specified in the gauge with the
+// WithKeys option.
 func (g *Int64Gauge) GetHandle(labels LabelSet) (h Int64GaugeHandle) {
 	h.Handle = g.getHandle(labels)
 	return
 }
 
+// Measurement creates a Measurement object to use with batch
+// recording.
 func (g *Float64Gauge) Measurement(value float64) Measurement {
 	return g.float64Measurement(value)
 }
 
+// Measurement creates a Measurement object to use with batch
+// recording.
 func (g *Int64Gauge) Measurement(value int64) Measurement {
 	return g.int64Measurement(value)
 }
 
+// Set sets the value of the gauge to the passed value. The labels
```

Comment:
Maybe rephrase it to "Set assigns the passed value to the value of the gauge".
Same for all 'Set' methods.